OCP-4.X/inventory.example

```diff
@@ -1,2 +1,81 @@
 [orchestration]
 # The server that contains the kubeconfig and oc command
+
+[all:vars]
+ORCHESTRATION_USER=root
+CLUSTER_USER=core
+GOPATH="/root/.go"
+DISABLE_SELINUX=False
+
+# Install vars
+OPENSHIFT_INSTALL_USER=core
```

Comment:
Can we collapse `CLUSTER_USER` and `OPENSHIFT_INSTALL_USER`? Or is it possible to need/require two different values for each of these vars?